monitoring/monit.py

```diff
@@ -119,7 +120,7 @@ def run_command(command):
         if module.check_mode:
             module.exit_json(changed=True)
         status = run_command('unmonitor')
-        if status in ['not monitored']:
+        if status in ['not monitored'] or 'unmonitor pending' in status:
```

Comment:
What's the string that you're seeing in status?  It seems like this test should be something more like one of these:
```
# We're able to give the exact string for everything:
if status in ('not monitored', 'unmonitor pernding'):
# We're not able to give the exact string for anything:
if [s for s in ('not monitored', 'unmonitor pending') if s in status]:
# We're able to give the exact string for one but not the other:
if status == 'not monitored' or 'unmonitor pending' in status:
```
A comment with examples of both strings that we're looking for might clarify the conditional for future people working on this.
```